test/v2/unit/coins/btc.js

```diff
@@ -81,19 +81,21 @@ describe('BTC:', function() {
     });
 
     it('should generate p2sh-wrapped segwit address', function() {
-      const generatedAddress = coin.generateAddress({ keychains, addressType: AbstractUtxoCoin.AddressTypes.P2SH_P2WSH });
-      const generatedTestAddress = testCoin.generateAddress({ keychains, addressType: AbstractUtxoCoin.AddressTypes.P2SH_P2WSH });
+      const addressType = Codes.UnspentTypeTcomb('p2shP2wsh');
+      const chain = Codes.forType(addressType)[Codes.PurposeTcomb('external')];
+      const generatedAddress = coin.generateAddress({ keychains, addressType, chain });
```

Comment:
If we're moving to use chain, why would we additionally use `addressType`?

Reply:
Because right now we default to using `p2sh` as the `addressType` so if we pass a `p2shP2wsh` chain with no `addressType` we will get an `AddressTypeChainMismatchError`.

Reply:
Okay